packages/protocol/contracts/common/interfaces/IEpochManager.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.5.13 <0.9.0;
+
+interface IEpochManager {
+  function initializeSystem(
+    uint256 firstEpochNumber,
+    uint256 firstEpochBlock,
+    // uint256 firstEpochTimestamp, // TODO: do we need END timestamp?
```

Comment:
Do we need the todo in interface ?

Reply:
it's still a WIP in #11191 so I havent removed it here.